addons/Connection/src/presenter.js

```diff
@@ -335,17 +411,121 @@ function AddonConnection_create() {
         };
     }
 
+    presenter.addDisabledElementsFromInitialValues = function () {
+        presenter.initialValues.forEach(function (initialValue) {
+           if (initialValue.isDisabled) {
+               presenter.disabledConnections.push({
+                   id1: initialValue.from,
+                   id2: initialValue.to
+               });
+           }
+        });
+    };
+
+
+
+    presenter.getInitialValues = function (model) {
+        var modelValidator = new ModelValidator();
+        var validated = modelValidator.validate(model, [
+            ModelValidators.List("initialConnections", [
+                ModelValidators.String("from", {default: "", trim: false}),
```

Comment:
Czy tu powinna być wartość domyślna?
Wypełnienie tylko jednego pola powinno być błędem.

Reply:
Jeżeli któraś z wartości jest pusta, to cały wpis jest uznawany jako nieważny.